### PR TITLE
OpenApi @@id support

### PR DESCRIPTION
**Pull request by thomassnielsen**
Companion to #1754 

@ymc9 how do I best update the baseline files with the extra model needed for new tests?

### CODE REVIEW

**Comment by ymc9** [comment]
> Companion to #1754
> 
> @ymc9 how do I best update the baseline files with the extra model needed for new tests?

The test cases print the location of the generated yaml files to console. I usually just copy/paste its content over to the "baseline" folder and diff to make sure the changes are expected 😄.

File: packages/plugins/openapi/tests/openapi-restful.test.ts [hunk 1]
```diff
@@ -55,12 +56,21 @@ model post_Item {
...
     })
 }
 
+model PostLike {
+    post post_Item @relation(fields: [postId], references: [id])
+    postId String
+    user User @relation(fields: [userId], references: [id])
+    userId String
+    @@id([postId, userId])
+}
```
[the remark this answers is not in the document]
Comment (thomassnielsen) [reply packages/plugins/openapi/tests/openapi-restful.test.ts:72]: Not needed for tests

File: packages/plugins/openapi/tests/openapi-restful.test.ts [hunk 2]
```diff
@@ -55,12 +56,21 @@ model post_Item {
...
     viewCount Int @default(0)
     notes String?
+    likes PostLike[]
 
     @@openapi.meta({
         tagDescription: 'Post-related operations'
     })
 }
 
+model PostLike {
```
Comment (ymc9) [inline packages/plugins/openapi/tests/openapi-restful.test.ts:66]: Just curious, do you know why the "PostLike" model doesn't show up as top level in the updated baseline files?

Comment (thomassnielsen) [reply packages/plugins/openapi/tests/openapi-restful.test.ts:66]: Hmm no. Could be that this requires some more work. I'll look into it.

Comment (ymc9) [reply packages/plugins/openapi/tests/openapi-restful.test.ts:66]: I see. That makes sense. Thank you!

**Review by ymc9 — APPROVED** [review]

**Comment by ymc9** [comment]
I'm merging it and will include the change in the upcoming 2.7.0 release.

**Pull request merged**